acceptable/tests/test_build_doubles.py

```diff
@@ -12,10 +18,19 @@
     Contains
 )
 
+
 from acceptable import _build_doubles
 
 
-class ExtractSchemasFromSourceTests(TestCase):
+class BuildDoubleTestCase(TestCase):
+
+    def setUp(self):
+        super().setUp()
+        if sys.version_info[0] == 2:
```

Comment:
Would be nice if these tests were consistent across the PR.  `from future.utils import PY2` and `if PY2:`?